lightning/src/util/events.rs

```diff
@@ -52,7 +52,8 @@ pub enum Event {
 	/// Indicates we've received money! Just gotta dig out that payment preimage and feed it to
 	/// ChannelManager::claim_funds to get it....
 	/// Note that if the preimage is not known or the amount paid is incorrect, you must call
-	/// ChannelManager::fail_htlc_backwards to free up resources for this HTLC.
+	/// ChannelManager::fail_htlc_backwards to free up resources for this HTLC before
+	/// LATENCY_GRACE_PERIOD_BLOCKS to avoid any channel-closure by onchain monitoring.
```

Comment:
I don't think this is correct - we don't know what our counterparty's criteria for channel-closure is.

Reply:
Precised its *our* onchain monitoring, not counterparty's one.

Reply:
But the sentence is all about when we *dont* have the preimage.